packages/frontend/native/nbstore/src/pool.rs

```diff
@@ -0,0 +1,264 @@
+use chrono::NaiveDateTime;
+use dashmap::{mapref::one::RefMut, DashMap, Entry};
+
+use super::{
+  error::{Error, Result},
+  storage::SqliteDocStorage,
+  DocClock, DocRecord, DocUpdate,
+};
+use crate::{Blob, ListedBlob, SetBlob};
+
+#[derive(Default)]
+pub struct SqliteDocStoragePool {
+  inner: DashMap<String, SqliteDocStorage>,
+}
+
+impl SqliteDocStoragePool {
```

Comment:
the pool should be a manager for `DocStorage` instead of involve all the apis `DocStorage` provided